Fabric.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "Fabric"
-  s.version      = "0.0.1"
+  s.version      = "1.2.5"
```

Comment:
@killian90 i thought that version is 3.0  or its just version of Crashlytics not Fabric ?